CHANGELOG.md

```diff
@@ -48,6 +48,9 @@
 
 #### Bug Fixes
 
+* Fix false positives on `no_grouping_extension` rule when using `where` clause.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

Reply:
Done. Thanks!